tasks/orchestrator.yml

```diff
@@ -16,3 +16,25 @@
   register: yaml_client_config_secret
   tags:
   - oc
+
+# Setup imagestream
+- name: copy imagestream
+  template:
+    src: osbs-buildroot-imagestream.yml.j2
+    dest: "{{ osbs_openshift_home }}/{{ inventory_hostname }}-{{ osbs_namespace }}-osbs-buildroot-imagestream.yml"
+  environment: "{{ osbs_environment }}"
+  register: yaml_imagestream
+  when: osbs_is_admin
+  tags:
+  - oc
+
+- name: import imagestream
```

Comment:
Let's rename this task. Import imagestream means something else :)

Reply:
does create imagestream sounds better?